tmt-app/src/Components/ResultsCards.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { Stack } from "@mui/material";
+import CustomCard from "./CustomCard";
+
+function ResultsCards() {
+  const dummyData = [
+    {
+      id: 0,
+      name: "Education",
+      desc: "testing",
+      icon: "emotional_intelligence",
+    },
+    {
+      id: 1,
+      name: "Economics and Business",
+      desc: "test fdasf afas fsadf dsf dsf edsf ds fsdaf sd f asfsd fea fgdsffdsf dfasdf ds fsdafsdf fgsdaf ased fasd few gasef sWEAFGASDFAS FDFSA DFA FSADF EAFSDASD FGREAASDFG EF GRasdfga gadfaefadf asf ",
+      icon: "chemistry",
+    },
+  ];
```

Comment:
Can we create a new file with mock data to export so we can clean up what we commit in these files?

Reply:
As well as use the actual data if that is available?

Reply:
The actual data needs some changes that would be done in a later ticket, but I could create a new file for the mock data.